### PR TITLE
Add UI commands to the post editor

### PR DESCRIPTION
**Pull request by youknowriad**
Related #51502 

## What?

Adds some simple UI related commands to the post editor. Similar commands have already been added to the site editor.

## Testing Instructions

1- Open the post editor
2- Run these commands

 - Toggle document settings
 - Toggle block inspector
 - Toggle distraction free
 - Toggle spotlight mode
 - Toggle top toolbar

### CODE REVIEW

File: packages/edit-post/src/hooks/commands/use-common-commands.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+				editPostStore.name
+			),
+			editorMode: select( editPostStore ).getEditorMode(),
+		} ),
+		[]
+	);
+	const { toggle } = useDispatch( preferencesStore );
+
+	useCommand( {
+		name: 'core/open-settings-sidebar',
```
Comment (ntsekouras) [inline packages/edit-post/src/hooks/commands/use-common-commands.js:37]: I noticed that we're inconsistent with the command names in edit post and site packages. Some times we have the names like here and others like: `core/edit-site/reset-global-styles`. I mean it makes sense to have the extra prefix for core commands, but probably we should keep it simple for commands that only load in each editor. WDYT?

**Review by ntsekouras — APPROVED** [review]
LGTM, thanks!

File: packages/edit-post/src/hooks/commands/use-common-commands.js (hunk 1 above)
Comment (youknowriad) [reply packages/edit-post/src/hooks/commands/use-common-commands.js:37]: yeah, after we add all the commands I think we should make a single PR to clean all the names. If not for allowing "unregistration" of commands from third-party plugins, I would remove these names entirely :P

**Comment by tellthemachines** [comment]
I just cherry-picked this PR to the release/16.1 branch to get it included in the next release: da17fa85dd

**Pull request merged**